architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -291,6 +351,19 @@ the xref:image-stream-object-definition[example above] would be
 ----
 ====
 
+
+[[image-stream-triggers]]
+=== Image Stream Triggers
+
+Image change triggers allow your build or deployment (via replication controller) to be automatically invoked when a new version of an upstream image is available.
```

Comment:
your build or deployment ~~(via replication controller)~~

That RC is not important in this context.